### ISSUE TITLE
Archiving fails on MacOS

### ISSUE THREAD

**Issue by jameswood** [ericaltendorf/plotman#380]
**Describe the bug**
Archiving fails when Plotman is run on MacOS

**To Reproduce**
Plotman uses rsync's `-no-compress` option but on a Mac we need to use `--compress-level=0`

**Expected behavior**
Finished jobs should be moved to the server via Rsync, instead they fail and retry forever

**System setup:**
 - OS: MacOS 11.2.3
 - Method of archiving: rsync sending to rsyncd on the Ubuntu Server

**Config**
per the Archiving guide

**Comment by altendky** [comment]
Thanks for the report.  Any chance you could give it a try and confirm it works for you?  You can follow the readme installation instructions but use `@altendky-patch-2` instead of `@main`.

**Comment by altendky** [comment]
I went ahead and cherry picked it to `@release/v0.3.1` as well (branch from which we will make the v0.3.1 release, hopefully soon).

**Comment by jameswood** [comment]
LEGEND - I'm running a manual rsync of all the overnight plots now, and I've installed `@release/v0.3.1`. It's plotting away happily now, and I'll enable archiving and report back once the manual rsync is finished.

Oh how I love to watch the little dots move across their timeline 😆

**Comment by jameswood** [comment]
OK SO mixed results: the command works but folder names aren't being slash-escaped. So, for `/Volumes/Plotting Temp 2` I get:

`rsync --bwlimit=80000 --compress-level=0 --remove-source-files -P /Volumes/Plotting Temp 2/plot-k32-2021-05-13-02-47-[snip].plot rsync://james@10.0.0.xxx:12000/chia/02`

which or course throws errors:

`rsync: link_stat "/Volumes/Plotting" failed: No such file or directory (2)
rsync: link_stat "/Users/James/Temp" failed: No such file or directory (2)
rsync: link_stat "/Users/James/2/plot-k32-2021-05-13-02-47-[snip].plot" failed: No such file or directory (2)`

I'll change the folder names, which work everywhere else in Plotman.

**Comment by altendky** [comment]
Yeah, handling spaces is on my list...  classic errors, what can I say.

**Comment by jameswood** [comment]
I mean, I knew the risks when I put the spaces there 😆

`ln -s` to the rescue! I didn't want to cancel the plots in progress. Should have another test result for you shortly. (3:6)

**Comment by jameswood** [comment]
Without spaces it works. Nice! Thank you! (And for Plotman in general, it brings me JOY)
